### PR TITLE
Add links to file bugs

### PR DESCRIPTION
**Pull request by staktrace**
Depends on https://github.com/mozsearch/mozsearch-mozilla/pull/27

### CODE REVIEW

**Comment by staktrace** [comment]
Currently deployed to https://kats.searchfox.org/mozilla-central/source

File: tools/src/bin/output-file.rs [hunk 1]
```diff
@@ -164,6 +197,18 @@ fn main() {
                 accel_key: None,
             });
         };
+        if let Some(ref bugzilla) = &bugzilla_data {
```
Comment (emilio) [inline tools/src/bin/output-file.rs]: nit: no need for the ampersand.

File: tools/src/bin/output-file.rs [hunk 2]
```diff
@@ -24,6 +24,31 @@ use languages::FormatAs;
...
+    json::Json::from_reader(&mut reader).ok()?.into_object()
+}
+
+/// For a given path, looks up the bugzilla product and component and
+/// returns it in a tuple if it could be found. The JSON data format
+/// comes from https://searchfox.org/mozilla-central/rev/47edbd91c43db6229cf32d1fc4bae9b325b9e2d0/python/mozbuild/mozbuild/frontend/mach_commands.py#209-223,243
+/// and is fairly straightforward.
+fn get_bugzilla_component<'a>(bugzilla_data: &'a json::Object, path: &str) -> Option<(&'a str, &'a str)> {
+    let mut path_obj = bugzilla_data.get("paths")?;
+    for path_component in path.split('/') {
```
Comment (emilio) [inline tools/src/bin/output-file.rs]: Does this need to be different for windows? I guess since we're skipping generated files anyway it may not matter.

Comment (staktrace) [reply tools/src/bin/output-file.rs]: No, these paths are all from the VM instance, so they will always have forward slashes (assuming that's what you meant by windows differences)

**Pull request merged**